WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/StartingInfo/PromptProgressBar/ProductCreationAIPromptProgressBarViewModel.swift

```diff
@@ -0,0 +1,133 @@
+                    "productCreationAIPromptProgressBar.main.inProgress",
+                    value: "Add more details. ",
+                    comment: "State when more details need to be added for the main prompt description suggestion in product creation with AI."
+                )
+            case .halfway:
+                return NSLocalizedString(
+                    "productCreationAIPromptProgressBar.main.halfway",
+                    value: "Getting better. ",
+                    comment: "State when the prompt description is improving for the main prompt description suggestion in product creation with AI."
+                )
+            case .almostDone:
+                return NSLocalizedString(
+                    "productCreationAIPromptProgressBar.main.almostDone",
+                    value: "Great prompt! ",
+                    comment: "State when the prompt description is great for the main prompt description suggestion in product creation with AI."
+                )
+            case .completed:
+                return NSLocalizedString(
+                    "productCreationAIPromptProgressBar.main.completed",
+                    value: "Great prompt! ",
+                    comment: "State when the prompt description is completed and great for the main prompt description suggestion in product creation with AI."
+                )
+            }
+        }
+
+        var secondaryDescription: String {
+            switch self {
+            case .start:
+                return NSLocalizedString(
+                    "productCreationAIPromptProgressBar.secondary.start",
+                    value: "Add your product’s name and key features, benefits, or details to help it get found online.",
+                    comment: "Initial state with instructions to add product name and key details for the prompt description in product creation with AI."
+                )
+            case .inProgress:
+                return NSLocalizedString(
+                    "productCreationAIPromptProgressBar.secondary.inProgress",
+                    value: "The more details you provide, the better your generated details will be.",
+                    comment: "State when more details will improve the generated content for the prompt description in product creation with AI."
+                )
+            case .halfway:
+                return NSLocalizedString(
+                    "productCreationAIPromptProgressBar.secondary.halfway",
+                    value: "Can you describe the fit and any distinctive features of the item?",
+                    comment: "State when the description should include fit and distinctive features for the prompt description in product creation with AI."
+                )
+            case .almostDone:
+                return NSLocalizedString(
+                    "productCreationAIPromptProgressBar.secondary.almostDone",
+                    value: "Where was it made?",
```

Comment:
❓ Should we use a more generic text here? We can't be sure that the user hasn't already answered this question in the keywords. What do you think?

Reply:
I edited it to `Mention additional relevant information or characteristics.`. Wdyt? 998206104e7e9837a9d1d242922697c779c6a6a7

Reply:
Sounds good to me. 👍